src/Paramore.Brighter.MessageScheduler.TickerQ/TickerQScheduler.cs

```diff
@@ -0,0 +1,199 @@
+﻿using System.Text.Json;
```

Comment:
You should include an MIT header in non-test files. You can use your own name as our CLA has you grant us rights to use, over ceding your rights. See: https://github.com/BrighterCommand/Brighter/blob/master/Contributing.md